src/astra_driver.cpp

```diff
@@ -1110,10 +1422,12 @@ void AstraDriver::initDevice()
     try
     {
       std::string device_URI = resolveDeviceURI(device_id_);
-      #if 0
+      #if 1
       if( device_URI == "" ) 
```

Comment:
```
if(device_URI.empty()) {}
```

Reply:
This would change the original code. (wish for the upstream merge)